deployment/hasura/metadata/databases/AerieMerlin/tables/public_simulated_activity_view.yaml

```diff
@@ -0,0 +1,50 @@
+table:
+  name: simulated_activity
+  schema: public
+object_relationships:
+- name: dataset
+  using:
+    manual_configuration:
+      remote_table:
+        name: dataset
+        schema: public
+      insertion_order: null
```

Comment:
Just curious what insertion_order means, and why it should be null - is that basically signifying that we should never insert into a view?

Reply:
I've honestly no idea, I pulled it in from pattern matching other metadata similarly

Reply:
removed these because no idea what they were about